pkg/apiserver/apiserver.go

```diff
@@ -277,13 +277,7 @@ func CleanupDeprecatedAPIServices(aggregatorClient clientset.Interface) error {
 	// deprecates a registered APIService, the APIService should be deleted,
 	// otherwise K8s will fail to delete an existing Namespace.
 	// Also check: https://github.com/antrea-io/antrea/issues/494
-	deprecatedAPIServices := []string{
-		"v1beta1.networking.antrea.tanzu.vmware.com",
-		"v1beta1.controlplane.antrea.tanzu.vmware.com",
-		"v1alpha1.stats.antrea.tanzu.vmware.com",
-		"v1beta1.system.antrea.tanzu.vmware.com",
-		"v1beta2.controlplane.antrea.tanzu.vmware.com",
-	}
+	deprecatedAPIServices := []string{}
```

Comment:
I guess it's OK to remove the function `CleanupDeprecatedAPIServices` and stop calling it?

Reply:
The multi-stage design of API determines that there will be stale APIs/versions need to be removed in the future, keep the code to avoid repeated efforts in the future.